packages/next-polyfill-nomodule/src/index.js

```diff
@@ -20,7 +37,10 @@ import 'core-js/features/number/is-nan'
 import 'core-js/features/number/is-safe-integer'
 import 'core-js/features/number/max-safe-integer'
 import 'core-js/features/number/min-safe-integer'
+import 'core-js/features/number/parse-float'
```

Comment:
This one looks fine as it's only not supported in legacy browsers:
https://caniuse.com/#feat=mdn-javascript_builtins_number_parsefloat
https://caniuse.com/#feat=mdn-api_htmlscriptelement_nomodule